internal/api/grpc/admin/smtp.go

```diff
@@ -128,3 +129,33 @@ func (s *Server) DeactivateSMTPConfig(ctx context.Context, req *admin_pb.Deactiv
 		Details: object.DomainToAddDetailsPb(result),
 	}, nil
 }
+
+func (s *Server) TestSMTPConfigById(ctx context.Context, req *admin_pb.TestSMTPConfigByIdRequest) (*admin_pb.TestSMTPConfigByIdResponse, error) {
+	instanceID := authz.GetInstance(ctx).InstanceID()
+	resourceOwner := instanceID // Will be replaced when orgs have smtp configs
+
+	err := s.command.TestSMTPConfigById(ctx, resourceOwner, req.Id, req.TestAddress)
```

Comment:
Can be combined with the 3 lines above, as it has to be changed anyway with logic changes.